### PR TITLE
TPC QC: Update of the Tracks task

### PR DESCRIPTION
**Pull request by stheckel**
o Changing p_T plots to logPtBinning
o Adding plots for neg. and pos. tracks, separately
o Adding ratios neg./pos. tracks, created at endOfCycle

### CODE REVIEW

File: Detectors/TPC/qc/src/Tracks.cxx [hunk 1]
```diff
@@ -28,21 +29,38 @@ using namespace o2::tpc::qc;
...
-  mHist2D.emplace_back("h2DEtaPhiNeg", "Negative tracks in eta vs. phi;phi;eta", 360, 0., 2 * M_PI, 400, -2., 2.);
-  mHist2D.emplace_back("h2DEtaPhiPos", "Positive tracks in eta vs. phi;phi;eta", 360, 0., 2 * M_PI, 400, -2., 2.);
+  const auto logPtBinning = helpers::makeLogBinning(30, 0.1, 20);
+
+  mHist1D.emplace_back("hNClustersBeforeCuts", "Number of clusters before cuts;# TPC clusters", 160, -0.5, 159.5); //| mHist1D[0]
+  mHist1D.emplace_back("hNClustersAfterCuts", "Number of clusters after cuts;# TPC clusters", 160, -0.5, 159.5);   //| mHist1D[1]
+  mHist1D.emplace_back("hEta", "Pseudorapidity;eta", 400, -2., 2.);                                                //| mHist1D[2]
+  mHist1D.emplace_back("hPhiAside", "Azimuthal angle, A side;phi", 360, 0., 2 * M_PI);                             //| mHist1D[3]
+  mHist1D.emplace_back("hPhiCside", "Azimuthal angle, C side;phi", 360, 0., 2 * M_PI);                             //| mHist1D[4]
+  //mHist1D.emplace_back("hPt", "Transverse momentum;p_T", 200, 0., 10.);                                               //| mHist1D[5]
```
Comment (wiechula) [inline Detectors/TPC/qc/src/Tracks.cxx]:
```suggestion
```

File: Detectors/TPC/qc/src/Tracks.cxx [hunk 2]
```diff
@@ -28,21 +29,38 @@ using namespace o2::tpc::qc;
...
+  mHist1D.emplace_back("hPhiCsideNeg", "Azimuthal angle, C side, neg. tracks;phi", 360, 0., 2 * M_PI);                  //| mHist1D[11]
+  mHist1D.emplace_back("hPhiCsidePos", "Azimuthal angle, C side, pos. tracks;phi", 360, 0., 2 * M_PI);                  //| mHist1D[12]
+  mHist1D.emplace_back("hPtNeg", "Transverse momentum, neg. tracks;p_T", logPtBinning.size() - 1, logPtBinning.data()); //| mHist1D[13]
+  mHist1D.emplace_back("hPtPos", "Transverse momentum, pos. tracks;p_T", logPtBinning.size() - 1, logPtBinning.data()); //| mHist1D[14]
+
+  mHist2D.emplace_back("h2DNClustersEta", "Number of clusters vs. eta;eta;# TPC clusters", 400, -2., 2., 160, -0.5, 159.5);                    //| mHist2D[0]
+  mHist2D.emplace_back("h2DNClustersPhiAside", "Number of clusters vs. phi, A side ;phi;# TPC clusters", 360, 0., 2 * M_PI, 160, -0.5, 159.5); //| mHist2D[1]
+  mHist2D.emplace_back("h2DNClustersPhiCside", "Number of clusters vs. phi, C side ;phi;# TPC clusters", 360, 0., 2 * M_PI, 160, -0.5, 159.5); //| mHist2D[2]
+  //mHist2D.emplace_back("h2DNClustersPt", "Number of clusters vs. p_T;p_T;# TPC clusters", 200, 0., 10., 160, -0.5, 159.5);                     //| mHist2D[3]
```
Comment (wiechula) [inline Detectors/TPC/qc/src/Tracks.cxx]:
```suggestion
```

**Review by wiechula — CHANGES_REQUESTED** [review]
@stheckel , thanks for the update! I would remove the commented hists, they are not needed. Please run clang-format afterwards.

File: Detectors/TPC/qc/src/Tracks.cxx (hunk 1 above)
Comment (stheckel) [reply Detectors/TPC/qc/src/Tracks.cxx]: Yes, agreed.

File: Detectors/TPC/qc/src/Tracks.cxx (hunk 2 above)
Comment (stheckel) [reply Detectors/TPC/qc/src/Tracks.cxx]: Yes.

**Review by wiechula — APPROVED** [review]

**Comment by wiechula** [comment]
@shahor02 , looks like the failing test in macos is unrelated. From my side this can be merged.

**Pull request merged**